platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/SynchronizedLogging.java

```diff
@@ -29,7 +29,7 @@
  * Provides logging services per thread.
  */
 public class SynchronizedLogging implements LoggingProvider {
-    private final ThreadLocal<ThreadLoggingServices> services = new ThreadLocal<ThreadLoggingServices>();
+    private final static ThreadLocal<ThreadLoggingServices> SERVICES = new ThreadLocal<>();
```

Comment:
is this naming some prone rule?


only picnic as this rule to fix once and globally.

- https://error-prone.picnic.tech/bugpatterns/ConstantNaming/